strax/dtypes.py

```diff
@@ -194,6 +194,8 @@ def peak_dtype(n_channels=100, n_sum_wv_samples=200, n_widths=11):
           'n_hits'), np.int32),
         (('Waveform data in PE/sample (not PE/ns!)',
           'data'), np.float32, n_sum_wv_samples),
+        (('Waveform data in PE/sample of the top array(not PE/ns!)',
+          'data_top'), np.float32, n_sum_wv_samples),
```

Comment:
@petergaemers could you look into reducing this precision to reduce the size as per Jelles suggestions as well as making this an optional field? Otherwise making it optional in the processing has little effect

Reply:
I thought that we agreed to start it from the full precision , as we will start with small number of runs and creation of top/bottom waveforms will be disabled by default. Other issue with reduced precision is the fact that we have undershoots to negative values etc, what can significantly affect peaks.

Reply:
> as well as making this an optional field

This at least should be easy to implement

>I thought that we agreed to start it from the full precision 

I don't remember converging to this convention. Also with this full precision you can have negative areas? But okay if you looked into it and it's not possible get this working, please go ahead